docs/migration.md

```diff
@@ -119,6 +119,9 @@ consider additional positioning prop support on a case-by-case basis.
   - The `v8` version of `@zendeskgarden/react-dropdowns` is no longer maintained and is
     renamed to `@zendeskgarden/react-dropdowns.legacy` in `v9`
 - `Menu`: value `auto` is no longer valid for the `fallbackPlacements` prop.
+- `Menu`: new `restoreFocus` prop (default: `true`) returns focus to trigger
+  after menu interaction. To keep the dropdown open on selection,
+  set `restoreFocus={false}` and manage focus manually.
```

Comment:
Not sure I understand this. Wouldn't the `isExpanded` controlled prop be the mechanism for keeping the menu open?

Reply:
I was referring to cases when a users chooses to keep the menu open after selecting an item within an `ItemGroup` of type `checkbox`. This [demo](https://66e49e3720ee94778c2a11a5--zendeskgarden-react-containers.netlify.app/?path=/story/packages-menu--controlled-managed-focus) showcases this pattern. 

It probably makes more sense to rephrase it as such:

```
`Menu`: new `restoreFocus` prop (default: `true`) returns focus to trigger
  after menu interaction. If keeping the dropdown open on selection is required,
  set `restoreFocus={false}` and manage focus manually.
```

What do you think?

Reply:
This still makes it sound like there is a causal relationship between keeping the menu open and setting `restoreFocus` to false.